dev-support/cicd/HowToRelease.md

```diff
@@ -26,16 +29,16 @@ svn ci -m "Add <your name>'s key"
 It's better to send a release plan email informing code freeze date and release date.
 
 ## 3. Clean up the JIRA
-Bulk update JIRA to unassign from this release all issues that are open non-blockers.
+Bulk update JIRA to unassigned from this release all issues that are open non-blockers.
```

Comment:
Let's update this to "Bulk update JIRA to move out non-blocker issues by setting the target version to the new release."